lang/pt.js

```diff
@@ -1,25 +1,25 @@
 /* eslint quotes: 0 */
 
 export default {
-	"components.alert.close": "Fechar Alerta",
+	"components.alert.close": "Fechar alerta",
 	"components.calendar.notSelected": "Não selecionado.",
 	"components.calendar.selected": "Selecionado.",
 	"components.calendar.show": "Mostrar {month}",
 	"components.dialog.close": "Fechar esta caixa de diálogo",
-	"components.input-date.clear": "Desmarcar",
-	"components.input-date.openInstructions": "",
-	"components.input-date.setToToday": "",
+	"components.input-date.clear": "Limpar",
```

Comment:
"Mark off" -> "To clean" 😆

Reply:
I switched Google to Portuguese and the clear button in their search bar uses "Limpar" so I'm good with this 👍